build/bootstrap.sh

```diff
@@ -93,29 +83,9 @@ function installBuildDependencies
     fi
 
     # Open-source library dependencies
-    # Boost libs for DO
+    # Boost libs
```

Comment:
Can just delete this comment now. Before it was to distinguish which were used for DO and which were used for cpprest